README.md

```diff
@@ -38,7 +38,7 @@ The YAML file tries to be self-explanatory.
 It tries to describe contributions thinking in possible future use of the data.
 
 Types are [CHAOSS Types of
-Contributions](https://github.com/chaoss/wg-common/blob/master/focus-areas/what/types-of-contributions.md).
+Contributions](https://github.com/chaoss/wg-common/blob/master/focus-areas/contributions/types-of-contributions.md).
 
```

Comment:
Interesting, they moved it a while ago, but after I linked it here https://github.com/chaoss/wg-common/commit/85afbbfeea3afed91dd35775b52c9c05e4bd1967 :sweat_smile: